package.json

```diff
@@ -1,18 +1,54 @@
 {
-	"name": "o-overlay",
-	"private": true,
+	"name": "@financial-times/o-overlay",
+	"version": "0.0.0",
+	"description": "Configurable custom overlay box that can be used to show overlay windows. The overlays can also be switched to display differently on small screens",
+	"keywords": [
+		"overlay",
+		"modal",
+		"interstitial",
+		"messaging",
+		"popover",
+		"notification",
+		"hover"
+	],
+	"homepage": "https://registry.origami.ft.com/components/o-overlay",
+	"bugs": {
+		"url": "https://registry.origami.ft.com/components/o-overlay"
+	},
+	"license": "MIT",
+	"type": "module",
+	"browser": "main.js",
+	"scripts": {
+		"start": "npx serve ./demos/local",
+		"build": "npm_config_yes=true npx \"origami-build-tools@^11\" install && npm_config_yes=true npx \"origami-build-tools@^11\" demo"
+	},
+	"dependencies": {
+		"ftdomdelegate": "^4.0.0"
+	},
 	"devDependencies": {
-		"eslint": "^7.8.1",
-		"eslint-config-origami-component": "^2.1.0",
-		"origami-ci-tools": "^2.0.2",
+		"eslint": "^7.24.0",
+		"eslint-config-origami-component": "^2.1.1",
 		"remark": "^13.0.0",
-		"remark-lint": "^8.0.0",
-		"remark-preset-lint-origami-component": "^1.1.3",
-		"stylelint": "^13.7.0",
+		"remark-lint": "^7.0.1",
+		"remark-preset-lint-origami-component": "prerelease",
```

Comment:
will need to update this to 2
```

Reply:
Thanks, I've updated this in [`c78fecb` (#274)](https://github.com/Financial-Times/o-overlay/pull/274/commits/c78fecbea1d805ce7a10129dc56c165f040c09ad)